spec/operations/result_spec.rb

```diff
@@ -255,7 +255,7 @@ def initialize(params, messages:)
         component: :contract,
         command: command_json,
         params: { id: 123, name: "Jon", lastname: "Snow" },
-        context: { record: "TraceableObject#1", object: "AnonymousObject" },
+        context: { record: "TraceableObject#1", object: '#<struct AnonymousObject name="Some name">' },
```

Comment:
it looks like we indeed should not use inspect

Reply:
Oh, okay, will revert it